src/india_api/internal/inputs/indiadb/client.py

```diff
@@ -370,6 +371,31 @@ def post_site_generation(
                 )
 
             generation_values_df = pd.DataFrame(generations)
+            capacity_factor = 1.1
```

Comment:
could you just make this an enviornment variable, like this https://github.com/openclimatefix/pv-site-api/blob/main/pv_site_api/main.py#L288C19-L288C80